src/extension.ts

```diff
@@ -39,11 +60,19 @@ export function activate(context: vsc.ExtensionContext): void {
     );
   }
 
-  const refreshViews = () => {
+  const refreshViews = (): void => {
     projectsProvider.refresh();
     codeLensProvider?.refresh();
   };
 
+  const refreshViewsDisposable = vsc.commands.registerCommand(
+    'statsig.refreshViews',
+    async (): Promise<void> => {
```

Comment:
whats the point of this being async, actually running this is prob gonna be in sync since you're calling this 1 at a time i think. (also, javascript doesnt support true async operations, like its fake lol)

Reply:
ohh good point